stix2/test/test_filesystem.py

```diff
@@ -40,6 +45,25 @@ def fs_sink():
     shutil.rmtree(os.path.join(FS_PATH, "campaign"), True)
 
 
+@pytest.fixture
```

Comment:
I don't know if you can use `scope=module` on this type of fixture (with yield), but it would potentially save a bunch of duplicate writing to disk, since the data is read-only. Maybe use a DataSource to make this more explicit?

Reply:
I'm actually starting to question whether it makes sense to have filesystem tests at all. None of the `relationships()` code is in the concrete data store classes, so it just seems to be duplicating tests.